ably/realtime_conn_test.go

```diff
@@ -120,8 +120,9 @@ func TestRealtimeConn_AuthError(t *testing.T) {
 	if err = ablytest.ConnWaiter(client, client.Connect, ably.ConnectionEventConnected).Wait(); err == nil {
 		t.Fatal("Connect(): want err != nil")
 	}
-	if state := client.Connection.State(); state != ably.ConnectionStateFailed {
-		t.Fatalf("want state=%s; got %s", ably.ConnectionStateFailed, state)
+	// Token errors are recoverable so we stay in DISCONNECTED and RTN14 kicks in
```

Comment:
This test also seems to be directed to the production env. I'd say it's too low-value to fix anyway. I'd suggest just removing it; we test (or will do) auth errors exhaustively elsewhere.

Reply:
done 825723188b172781e15028a6bdfac40b9f26198a